### DISCUSSION TITLE
I get this error when trying to commit from github desktop

### DISCUSSION THREAD

**Discussion by itayron10** [community/community#67850] [Actions] [answered]
### Select Topic Area

Question

### Body

when I clone a repository in the git hub desktop and try to commit something I get this error and it didn't happen before just now with my new repository 

Error: cannot spawn gpg: No such file or directory error: gpg failed to sign the data fatal: failed to write commit object

**Comment by seyLu** [comment, marked as answer]
#### Hi @itayron10,

Did you intend to enable signed commits in your git config?

If not, just disable gpg signing:
```bash
# this is the default behaviour
git config --global commit.gpgsign false
```

**Reply by itayron10** [reply]
Thank you it worked!
